src/Executor/Runner/Docker.php

```diff
@@ -598,11 +590,14 @@ public function createRuntime(
                 'duration' => $duration,
             ]);
 
-            $activeRuntime = $this->activeRuntimes->get($runtimeName);
-            $activeRuntime['updated'] = \microtime(true);
-            $activeRuntime['status'] = 'Up ' . \round($duration, 2) . 's';
-            $activeRuntime['initialised'] = 1;
-            $this->activeRuntimes->set($runtimeName, $activeRuntime);
+            $runtime = $this->runtimes->get($runtimeName);
+            if ($runtime !== null) {
+                $runtime->updated = \microtime(true);
+                $runtime->status = 'Up ' . \round($duration, 2) . 's';
+                $runtime->initialised = 1;
+
+                $this->runtimes->set($runtimeName, $runtime);
+            }
```

Comment:
get&set are not atomic, but I think swoole tables can be atomic. maybe some kind of `update` or `setProperty`. Let's chcek, could be nice improvement here